components/engine_traits/src/tablet.rs

```diff
@@ -178,7 +177,7 @@ impl<EK: Clone + Send + Sync> TabletFactory<EK> for SingletonFactory<EK> {
 struct TabletRegistryInner<EK> {
     // region_id, suffix -> tablet
     tablets: Mutex<HashMap<u64, CachedTablet<EK>>>,
-    tombstone: Mutex<Vec<(u64, u64)>>,
+    tombstone: Mutex<Vec<EK>>,
```

Comment:
If what's needed is just Path, then there seems to be no reason to store the actual tablet here. Drop it ahead of time should make cleanup faster.

Reply:
I think the overhead of dropping a db is not well-defined, we should probably do it in the background in case it waits for some blocking IOs.

Reply:
Then I think we can introduce two list here. One for tombstone EK, one for tombstone region id and tablet suffix. The first one is to call `PauseBackgroundWork` in the worker, the second is to trigger actual delete.